Makefile

```diff
@@ -1,5 +1,5 @@
 all: tools dependencies xcodeproj-httpservermock templates
-.PHONY : examples tools
+.PHONY : examples tools newversion
```

Comment:
this is probably not needed, this `phony` thing is needed for the folders with those names such as `examples` and `tools`

Reply:
ok, i will clean also the examples one, since there is not target for examples anymore